docs/getting_started.md

```diff
@@ -362,8 +352,16 @@ The project line can optionally use `SKBUILD_PROJECT_NAME` and
 your `pyproject.toml`. You should specify exactly what language you use to keep
 CMake from searching for both `C` and `CXX` compilers (the default).
 
-You'll need to handle the generation of files by Cython directly at the moment.
-A helper (similar to scikti-build classic) might be added in the future.
```

Comment:
FYI, it was a type from before, so not your fault, just nice to fix while looking at it. :)